pkg/server/server.go

```diff
@@ -56,17 +63,28 @@ func redirectMiddleware(exactPath string, redirectTo string) gin.HandlerFunc {
 }
 
 func CreateKHIServer(engine *gin.Engine, inspectionServer *coreinspection.InspectionTaskServer, serverConfig *ServerConfig) *gin.Engine {
-	appHtmlPath := path.Join(serverConfig.StaticFolderPath, "/index.html")
-
 	basePathWithoutTrailingSlash := strings.TrimSuffix(serverConfig.ServerBasePath, "/")
 	engine.Use(redirectMiddleware(basePathWithoutTrailingSlash+"/", basePathWithoutTrailingSlash+"/session/0")) // Request for `/` shouldn't be handled by `static.Serve`, redirect `/session/0` to be handled by patternToString
-	engine.Use(static.Serve(basePathWithoutTrailingSlash+"/", static.LocalFile(serverConfig.StaticFolderPath, false)))
+
+	// By default, use the embedded web files. If the static folder path is set, use the local file system.
+	slog.Debug("Using embedded static web files.")
```

Comment:
It's just a debug logging, but it can log `Using embedded static web files.` even when user gives non empty `serverConfig.StaticFolderPath`.

Reply:
Good catch! Fixed in 969b6ff843f4ded38318e1c4ae47fb274d8762f9